addons/context_actions/CfgContext.hpp

```diff
@@ -137,6 +137,31 @@ class EGVAR(context_menu,actions) {
             statement = QUOTE([ARR_2(_selectedObjects,'DOWN')] call FUNC(setStance));
         };
     };
+    class Captives {
+        displayName = "$STR_ACE_Captives_DisplayName";
+        condition = QUOTE( \
+            isClass (configFile >> 'CfgPatches' >> 'ace_captives') \
+            && {isClass (configFile >> 'CfgPatches' >> 'ace_zeus')} \
+            && {_selectedObjects findIf {_x isKindOf 'CAManBase' && {alive _x}} != -1});
```

Comment:
Move this to the condition the `ToggleCaptive` action below's condition. It will be checked then before the actions statement is executed.